dipy/viz/horizon/util.py

```diff
@@ -63,3 +63,32 @@ def check_img_dtype(images):
             warnings.warn(msg.format(idx + 1))
 
     return valid_images
+
+
+def is_binary_image(data, unique_points=100):
+    """Check if an image is binary image.
+
+    Parameters
+    ----------
+    data : ndarray
+    unique_points : int, optional
+        number of points to sample the check, by default 100
+
+    Returns
+    -------
+    boolean
+        Whether the image is binary or not
+    """
+    indices = []
+
+    for dim in data.shape:
+        indices.append(
+            np.random.choice(
```

Comment:
Numpy suggest the migration from `numpy.random` to `numpy.random.Generator`. All the codebase have been migrated. Can you do it here also? 

Thank you